resources/lib/jellyfin/core/http.py

```diff
@@ -100,7 +100,7 @@ def request(self, data, session=None):
                     continue
 
                 LOG.error(error)
-                self.client['callback']("ServerUnreachable", {'ServerId': self.config['auth.server-id']})
+                self.client.callback("ServerUnreachable", {'ServerId': self.config.data['auth.server-id']})
```

Comment:
`server-id` was replaced with `server_id` in multiple places in other files above.  Most of the project uses the underscore version, so I think we should finish these re-namings so everything is consistent.